css/main.css

```diff
@@ -0,0 +1,510 @@
+  .colors-rigth-project,
+  .colors-left-project,
+  .colors-left-about {
+    display: none;
+  }
+}
+
+@keyframes color {
+    from {margin-left: 0;}
+    to {margin-left: 60rem}
+    
+}
+
+@keyframes color2 {
+    from {margin-right: 0;}
+    to {margin-right: 60rem}
+    
+  }
+
+/* End Components */
+
+.ToDoListContainer {
+  position: relative;
+  display: flex;
+  background-color: #a18aff;
+  justify-content: center;
+  gap: 1rem;
+  border-radius: 5rem;
+  border-right: 15px solid #fff;
+  width: 100%;
+  height: 90vh;
+  margin-top: 5vh;
+}
+/* Start LeftBar */
+.leftContainer {
+  background-color: #ffffff;
+  padding: 2rem;
+  flex: 1.2;
+  min-height: 100%;
+  border-radius: 5rem;
+  border-top-right-radius: 7rem;
+  border-bottom-right-radius: 7rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.ProfileDetails {
+  display: flex;
+  justify-content: center;
+  gap: 1rem;
+  align-items: center;
+}
+
+.imageProfile {
+  width: 90px;
+  height: 80px;
+  border: #7760cb;
+}
+
+.nameProfile h3 {
+  margin-bottom: 0;
+  padding-bottom: 0;
+  font-weight: 400;
+  color: #a9a9a9;
+}
+.nameProfile h2 {
+  margin-top: 5px;
+  padding-top: 0;  
+  font-size:22px;
+  color:#bd93f4;
+  font-weight: 600;
+
+}
+
+.title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap:10px;
+}
+.title h1 {
+  font-size: 22px;
+  margin-left: 20px;
+  color: #6d6c6c;
+  font-weight: 600;
+}
+.title i {
+  color:#bd93f4;
+  font-size: 22px;
+}
+.fa-file-code {
+  font-size: 2rem;
+
+}
+.confirmAddTask{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 1rem;
+    margin-bottom: 5rem;
+    gap: 10%;
+
+}
+.ChoosType{
+    width: 50%;
+    background-color: #c2b6f5;
+    color:#fff;
+    border: none;
+    padding: .4rem;
+    border-radius: 1rem;
+}
+
+.buttonAddTask{
+    border: none;
+    width: 40%;
+    height: 2.5rem;
+    border-radius: 2rem;
+    background-color: #7258e4;
+    color: #fff;
+    cursor: pointer;
+    transition: all .5s ease-in-out;
+
+}
+.buttonAddTask:hover {
+    background-color: rgb(94, 80, 155);
+    border: none;
+    transform: scale(1.1);
+
+}.TodayTask{
+  margin-top: 3.5rem;
+}
+.DetailsTodayTask {
+    margin-left: 30px;
+}
+.DetailsTodayTaskInsideYallow::before {
+  content: "  ";
+  display: inline-block;
+  flex-direction: row;
+  width: 15px;
+  height: 15px;
+  border-radius: 50%;
+  background-color: #ffca0f;
+  margin-right: 2rem;
+}
+
+.icons i {
+  color:#bd93f4;
+}
+
+
+.DetailsTodayTaskInsideBlue::before {
+  content: "  ";
+  display: inline-block;
+  flex-direction: row;
+  width: 15px;
+  height: 15px;
+  border-radius: 50%;
+  background-color: #3fd2f6;
+  margin-right: 2rem;
+}
+.DetailsTodayTaskInsideRed::before {
+  content: "  ";
+  display: inline-block;
+  flex-direction: row;
+  width: 15px;
+  height: 15px;
+  border-radius: 50%;
+  background-color: #fa9aad;
+  margin-right: 2rem;
+}
```

Comment:
.DetailsTodayTaskInsideBlue::before, .DetailsTodayTaskInsideRed::before 
the styles for these 2 selectors are the same expect the color you can put the shared styles together , and split the colors based on the class name
.DetailsTodayTaskInsideBlue::before,
.DetailsTodayTaskInsideRed::before {
   //shared styles
}